src/main/java/org/kuali/kfs/fp/document/web/struts/DisbursementVoucherAction.java

```diff
@@ -202,7 +203,7 @@ public ActionForward printDisbursementVoucherCoverSheet(ActionMapping mapping, A
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         DisbursementVoucherCoverSheetService coverSheetService = SpringContext.getBean(DisbursementVoucherCoverSheetService.class);
 
-        coverSheetService.generateDisbursementVoucherCoverSheet(directory, DisbursementVoucherConstants.DV_COVER_SHEET_TEMPLATE_NM, document, baos);
+        coverSheetService.generateDisbursementVoucherCoverSheet(document, baos);

```

Comment:
It probably should (and I'll mention it to the product team) but it's still there in base code and since this is a file we overlay, I think we should keep it as close to base as possible. Thanks.